### PR TITLE
18 add a logger

### PR DESCRIPTION
**Pull request by cpaniaguam**
#18

### CODE REVIEW

File: workflow/scripts/ice-floe-tracker.jl [hunk 1]
```diff
@@ -21,27 +51,35 @@ function main(args)
...
-    add_arg_table!(settings["landmask"], landmask_cloudmask_args...)
+    mkcli!(settings, command_common_args)
 
     parsed_args = parse_args(args, settings; as_symbols=true)
 
     command = parsed_args[:_COMMAND_]
     command_args = parsed_args[command]
     command_func = getfield(IFTPipeline, Symbol(command))
-    @time command_func(; command_args...)
+    logoption = command_args[:log]
```
Comment (tdivoll) [inline workflow/scripts/ice-floe-tracker.jl:73]: will the logger report back on the time for each task to run? I'm assuming it replaces the @time macro used before, but just wanted to c=double check!

**Review by tdivoll — APPROVED** [review]
Looks good! I just wanted to check that the logger will time everything in addition to any debug info?

File: workflow/scripts/ice-floe-tracker.jl (hunk 1 above)
Comment (cpaniaguam) [reply workflow/scripts/ice-floe-tracker.jl:73]: Good catch! That didn't merge right. Fixed!

**Review by tdivoll — APPROVED** [review]
Nice! Glad I was able to catch something

**Pull request merged**